spec/modules-spec.js

```diff
@@ -129,3 +131,25 @@ describe('Module._nodeModulePaths', function () {
     })
   })
 })
+
+describe('require', () => {
+  describe('when loaded URL is not file: protocol', () => {
+    let w
+
+    beforeEach(() => {
+      w = new BrowserWindow({
+        show: false
+      })
+    })
+
+    it('searches for module under app directory', async () => {
+      w.loadURL('about:blank')
+      const result = await w.webContents.executeJavaScript('typeof require("q").when')
+      assert.equal(result, 'function')
+    })
+
+    afterEach(() => {
```

Comment:
`afterEach` blocks usually go after the `beforeEach` but before the `it` blocks, mind moving it there?